content/en/blog/2022/SIG-Storage Spotlight.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: blog
+title: "Spotlight on SIG Storage"
+date: 2022-08-02
```

Comment:
This is likely to follow close on the SIG Docs spotlight article. How about we defer this for a week or so (more if folks want to)?

Reply:
Not a problem to me, whenever it's suitable.